test/message/test_runner_hooks.out

```diff
@@ -55,6 +55,16 @@ not ok 2 - before throws
   failureType: 'hookFailed'
   error: 'failed running before hook'
   code: 'ERR_TEST_FAILURE'
+  stack: |-
+    Object.<anonymous> (*)
+    TestHook.runInAsyncScope (*)
+    TestHook.run (*)
+    TestHook.run (*)
```

Comment:
Do we need to bother with the content of the stack? Non-blocking of course, but I'm curious why we would test it here as I think we don't anywhere else in the test runner tests.
```suggestion
    *
    *
    *
    *
```

Reply:
not sure it is really needed, I figured we can check the stack contains the "important" parts indicating the flow of calling a hook - but for consisnacy I will follow the standard